perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -117,6 +117,7 @@
 
 DEFAULT_BOTO_LOCATION = '~/.boto'
 BOTO_LIB_VERSION = 'boto_lib_version'
```

Comment:
Add a comment here to indicate we use boto for GCS and boto3 for S3

<do you have any plan to update to boto3 for GCS?>